packages/esm-appointments-app/src/hooks/useVisits.ts

```diff
@@ -5,15 +5,18 @@ import { useAppointmentDate } from '../helpers';
 
 /**
  * Custom hook to fetch visits from the OpenMRS REST API.
+ * This fetches all visits that started on the currently selected date and are associated with the current session location.
  * @returns An object containing the visits, isLoading flag, and error message.
  */
 export const useVisits = () => {
   const { currentAppointmentDate } = useAppointmentDate();
   const session = useSession();
 
-  const visitsUrl = `${restBaseUrl}/visit?includeInactive=true&v=custom:(uuid,patient:(uuid,identifiers:(identifier,uuid),person:(age,display,gender,uuid)),visitType:(uuid,name,display),location:(uuid,name,display),startDatetime,stopDatetime)&fromStartDate=${dayjs(
+  const visitsUrl = `${restBaseUrl}/visit?includeInactive=true&includeParentLocations=true&v=custom:(uuid,patient:(uuid,identifiers:(identifier,uuid),person:(age,display,gender,uuid)),visitType:(uuid,name,display),location:(uuid,name,display),startDatetime,stopDatetime)&fromStartDate=${dayjs(
     currentAppointmentDate,
-  ).format('YYYY-MM-DD')}&location=${session?.sessionLocation?.uuid}`;
+  ).format('YYYY-MM-DD')}&toStartDate=${dayjs(currentAppointmentDate)
+    .add(1, 'day')
+    .format('YYYY-MM-DD')}&location=${session?.sessionLocation?.uuid}`;
```

Comment:
This is the main change, it adds a parameters to include parent locations and set a max start date when searching for visits.

Reply:
Assuming you make the change I suggested in the REST WS module, you should be able to set the `toStartDate` to the same value you set the `fromStartDate` to, and it will return all visits that started on that date.  Otherwise, won't this also return visits from the next date at midnight, which isn't really what you want?

Reply:
Good point re: visits that start at midnight.